tests/assertions/ocp4/rhcos4-high-4.12.yml

```diff
@@ -3,11 +3,11 @@ rule_results:
     default_result: PASS
     result_after_remediation: PASS
   e2e-high-master-audit-rules-dac-modification-chmod:
-    default_result: FAIL
-    result_after_remediation: PASS
+    default_result: FAIL OR NOT-APPLICABLE
```

Comment:
I wonder if it doesn't like the uppercase `OR`.


```
helpers.go:879: E2E-ERROR: The e2e YAML for rule 'e2e-moderate-master-audit-rules-dac-modification-lchown' is malformed: {FAIL OR NOT-APPLICABLE PASS OR NOT-APPLICABLE <nil>} . Got error: Error parsing result evaluator: result eval syntax error: cannot merge leaf and operand
```

Reply:
> I wonder if it doesn't like the uppercase `OR`.
> 
> ```
> helpers.go:879: E2E-ERROR: The e2e YAML for rule 'e2e-moderate-master-audit-rules-dac-modification-lchown' is malformed: {FAIL OR NOT-APPLICABLE PASS OR NOT-APPLICABLE <nil>} . Got error: Error parsing result evaluator: result eval syntax error: cannot merge leaf and operand
> ```

that's a good catch, let me try that out